com.unity.render-pipelines.core/Runtime/Debugging/IVolumeDebugSettings.cs

```diff
@@ -14,11 +14,14 @@ public interface IVolumeDebugSettings
         int selectedComponent { get; set; }
 
         /// <summary>Current camera to debug.</summary>
-        Camera selectedCamera { get; set; }
+        Camera selectedCamera { get; }
 
         /// <summary>Returns the collection of registered cameras.</summary>
         IEnumerable<Camera> cameras { get; }
 
+        /// <summary>Selected camera index.</summary>
+        int selectedCameraIndex { get; set; }
```

Comment:
this reverts the removal of the property, but what about the field `m_SelectedCameraIndex` ? It is still a breaking change but it looks like there's no way to preserve it.

Reply:
recovered